src/components/forms/AddSponsorKeyForm.js

```diff
@@ -148,7 +152,7 @@ export const AddSponsorKeyForm = forwardRef(({ onSuccess }, ref) => {
                         <TextField
                           type="text"
                           name="nextSkylink"
-                          placeholder={`Paste next skylink here`}
+                          placeholder={`Paste${skylinks.length > 0 ? " next" : " first"} skylink here`}
```

Comment:
nitty nit: putting the space in the string you append in the short if is a creating good preconditions for a missing space later. It makes sense to do that if only one of the cases has a leading space but this is not what we have here.

Reply:
```suggestion
                          placeholder={`Paste ${skylinks.length > 0 ? "next" : "first"} skylink here`}
```